### PR TITLE
add import/export command and test

### LINKED ISSUE #6971: add exportMessage and importMessage command for easy edit file
### Is Your Feature Request Related to a Problem?

1. migrate message across env:  we switch our system from aws to aliyun , some message still not consumed in aws , we should migrate message from aws to aliyun . 
2. fix some wrong message: after deploy some bug , the message format is not correct , we can export message --> modify message --> import messasge 
3. search message body in lots of message: after export message with json/text format , we could easily search some keywords

### Describe the Solution You'd Like

add exportMessage and importMessage command 

### Describe Alternatives You've Considered

add exportMessage and importMessage command 

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by carlvine500**
### Which Issue(s) This PR Fixes

Fixes #6971

### Brief Description

1. migrate message across env: we switch our system from aws to aliyun , some message still not consumed in aws , we should migrate message from aws to aliyun ==> just use exportMessage and importMessage .
2. fix some wrong message: after deploy some bug , the message format is not correct ==> we can exportMessage with json body --> modify message  body --> importMessasge 
3. trace bug in lots of message body  ==> after export message with json/text format  ,we could easily search some keywords to view message detail , such as `grep keyword messageFile ` 

### How Did You Test This Change?

1. exportMessage: mock broker , fetch some messages and export to file , at the same time , show result 100% on system.out
2. importMessage: mock broker , read a message file and send message to broker , at the same time , show result 100% on system.out

### example usage
WeChatc0e14e48d1e5c6ef1ee135a5141a74bf

### CODE REVIEW

**Review by mxsm — COMMENTED** [review]
@carlvine500  Please check code sytle and fix it

**Comment by carlvine500** [comment]
> @carlvine500 Please check code sytle and fix it

ok

**Comment by carlvine500** [comment]
@mxsm bazel-compile still failed , can I ignore it ?

**Comment by mxsm** [comment]
> @mxsm bazel-compile still failed , can I ignore it ?

Yes, you can ignore it

File: tools/src/main/java/org/apache/rocketmq/tools/command/message/ExportMessageCommand.java [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+                }
+
+                if (commandLine.hasOption('e')) {
+                    String timestampStr = commandLine.getOptionValue('e').trim();
+                    long timeValue = timestampFormat(timestampStr);
+                    maxOffset = defaultMQPullConsumer.searchOffset(mq, timeValue);
+                }
+
+                System.out.printf("export %s minOffset=%s, maxOffset=%s%n", minOffset, maxOffset, mq);
+                try (BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(new FileOutputStream(queueFile), charsetName))) {
```
Comment (ferrirW) [inline tools/src/main/java/org/apache/rocketmq/tools/command/message/ExportMessageCommand.java:164]: It is recommended to add a restriction strategy, such as limiting the latest timestamp returned or the maximum number of returned messages.

File: tools/src/main/java/org/apache/rocketmq/tools/command/message/ExportMessageCommand.java [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+                    READQ:
+                    for (long offset = minOffset; offset < maxOffset; ) {
```
Comment (ferrirW) [inline tools/src/main/java/org/apache/rocketmq/tools/command/message/ExportMessageCommand.java:166]: It is recommended to be asynchronous, and to support breakpoint resume from commited offset.

**Review by ferrirW — DISMISSED** [review]

**Review by carlvine500 — COMMENTED** [review]
it's fixed in latest version

**Comment by carlvine500** [comment]
@mxsm    @ferrirW please check it again .

**Pull request closed without merging**